### ISSUE TITLE
Make Stripe Payment Processor extension to work as stripe recomendation

### ISSUE THREAD

**Issue by sivaunq** [drastik/com.drastikbydesign.stripe#148]
Stripe has used tokenized input to avoid sending credit card information to merchant server and as per stripe documentation, 'name' attribute should not be used for credit card related fields but stripe civicrm extension submitting CC field value to our server.
Is it possible to not sending CC field value to our server?

**Comment by drastik** [comment]
It removes the sensitive CC field values before submitting to CiviCRM.
I think that is the question?  So, CC card values should not be transferred to your server.

**Comment by sivaunq** [comment]
Am using version 1.9.1. Currently the sensitive CC field values being transferred to our server but it should not happen.

**Comment by laryn** [comment]
My understanding is that 1.9.1 should not be used anymore, but rather download from 4.6-dev or 4.7-dev depending which version of CiviCRM you are using -- is that right @drastik ? _In my opinion it would be wise to start doing periodic releases (e.g. 4.6-1.0 or 4.7-1.0) to make debugging simpler. As it stands it is hard to tell which version anybody is using in their bug reports._

**Comment by drastik** [comment]
Yes, that is a good idea.  I've been debating how to address that.

I like the `4.7-1.0` format.

**Comment by drastik** [comment]
I think the CiviCRM extension guidelines call for the `x.x.x` format though.

**Comment by drastik** [comment]
Okay actually I found the page.  Here is the excerpt:

```


version of this extension (used for upgrade detection and used to sort available releases for automated distribution)

Valid version formats include:

'1', '1.1', '1.2.3.4', '1.2-3', '1.2.alpha2', '1.2.rc2', '2012-01-01-1', '2012-01-01', 'r456', 'r5000'
```

Source: https://wiki.civicrm.org/confluence/display/CRMDOC/Extension+Reference

**Comment by drastik** [comment]
So it looks like the 4th example in there makes the idea valid (`1.2-3`) as `4.7-1.0`

**Comment by laryn** [comment]
There are a number of questions in the issues queue just asking about releases/versions which could be closed when these releases are made.

**Comment by sivaunq** [comment]
Now i got the correct version. Thanks @laryn
